api/test/integration/types.test.js

```diff
@@ -0,0 +1,34 @@
+const { describe, test } = require('mocha')
+const { expect } = require('chai')
+
+const { setupApi, runProcess, utf8ToUint8Array, getLastTokenId, getToken } = require('../helper/routes')
+
+const USER_ALICE_TOKEN = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY'
+const METADATA_KEY_LENGTH = 32
+const METADATA_VALUE_LITERAL_LENGTH = 32
+
+describe('types', function () {
+  const context = {}
+
+  before(async function () {
+    await setupApi(context)
+  })
+
+  describe('health check', function () {
+    test('mint token', async function () {
+      const lastTokenId = await getLastTokenId(context)
+      const roles = new Map(Object.entries({ 0: USER_ALICE_TOKEN }))
```

Comment:
This could be exported as fixture and used across all tests or some helper method for different scenarios e.g. errors